### ISSUE TITLE
Error: null_resource.npk_component: : invalid or unknown key: environment

### ISSUE THREAD

**Issue by phatwila** [Coalfire-Research/npk#74]
Hello,
Getting this error when attempting to setup the selfhost wordlist. I will say standard NPK works without issue:

```
root@PC-76223:/home/npk/terraform-selfhost# vi npk-selfhost.auto.tfvars
root@PC-76223:/home/npk/terraform-selfhost# terraform apply
Plugin reinitialization required. Please run "terraform init".
Reason: Could not satisfy plugin requirements.

Plugins are external binaries that Terraform uses to access and manipulate
resources. The configuration provided requires plugins which can't be located,
don't satisfy the version constraints, or are otherwise incompatible.

5 error(s) occurred:

* provider.archive: no suitable version installed
  version requirements: "(any version)"
  versions installed: none
* provider.aws: no suitable version installed
  version requirements: "(any version)"
  versions installed: none
* provider.null: no suitable version installed
  version requirements: "(any version)"
  versions installed: none
* provider.template: no suitable version installed
  version requirements: "(any version)"
  versions installed: none
* provider.local: no suitable version installed
  version requirements: "(any version)"
  versions installed: none

Terraform automatically discovers provider requirements from your
configuration, including providers used in child modules. To see the
requirements and constraints from each module, run "terraform providers".


Error: error satisfying plugin requirements

root@PC-76223:/home/npk/terraform-selfhost# terraform init

Initializing provider plugins...
- Checking for available provider plugins on https://releases.hashicorp.com...
- Downloading plugin for provider "null" (2.1.2)...
- Downloading plugin for provider "template" (2.1.2)...
- Downloading plugin for provider "local" (1.4.0)...
- Downloading plugin for provider "aws" (2.46.0)...
- Downloading plugin for provider "archive" (1.3.0)...

The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.archive: version = "~> 1.3"
* provider.aws: version = "~> 2.46"
* provider.local: version = "~> 1.4"
* provider.null: version = "~> 2.1"
* provider.template: version = "~> 2.1"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
root@PC-76223:/home/npk/terraform-selfhost# terraform apply

Error: null_resource.npk_component: : invalid or unknown key: environment

root@PC-76223:/home/npk/terraform-selfhost# terraform -v
Terraform v0.11.0
```
Looks to be the same problem as #27 and also happens for both a normal user and root user account.

**Comment by doyler** [comment]
I'm seeing the same issue as this, also with v0.11.0

**Comment by romperchomper** [comment]
> Error: null_resource.npk_component: : invalid or unknown key: environment

Also having this issue. -v 11.0 as well.

**Comment by gchetrick** [comment]
This is fixed if you upgrade terraform to a later 0.11 version. I'm using 
Terraform v0.11.15-oci
